src/components/data-management/SamplesTable.jsx

```diff
@@ -221,19 +235,14 @@ const SamplesTable = forwardRef((props, ref) => {
   };
 
   useEffect(() => {
-    if (!samples[activeProject.samples[0]]) {
+    if (activeProject.samples.length === 0) {
```

Comment:
This if is checking for something differnet than it was before, is that ok? 
Before: the first sample has sample data in redux.
After: the active project has sample ids

Reply:
The previous if statement checks if the first `sampleUuid` in `projects.samples` exists in `samples`. That first `sampleUuid` might not exist in `samples`, but the remaining `sampleUuid`s might exist, and we should still show those samples. This if statement checks if there really is no sample registered to the project, and if that's the case, show an empty table.

Reply:
I am not sure I understand -- why do you need to change this?

Reply:
I discovered this when I was writing a test. In the test, I have a `projects` item with 3 samples in an array: A, B, C. The `samples` redux object initiallly have 3 samples too : A, B and C. In the test, I deleted A from the samples. This means that `samples` now only have data for 2 samples : B and C. In this case, we would still like to show the samples B and C, and show A as missing. However, with the previous statement, the table returns empty, because it specifically checks for the existence of the first sample, not if the projects have samples.

Reply:
it is clear for me now, thank you for the example and explanation!